recipes/Cm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cm" %}
+{% set version = "9.0.0" %}
+{% set versions = version.split('.') %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: ffcd65f5ff779db07bc2fbe982da6277ffa495aa6d66300c3194bd4a344311a4
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - meson >=0.49
+    - ninja
+    - bison
+    - flex
+    - pkg-config
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - test -x ${PREFIX}/bin/cm  # [unix]
+    - test -x ${PREFIX}/bin/NameServer  # [unix]
+    - test -f ${PREFIX}/lib/libcm.so  # [unix]
+    - pkg-config --print-errors --modversion cm
+
+about:
+  home: https://git.ligo.org/virgo/virgoapp/Cm/
+  dev_url: https://git.ligo.org/virgo/virgoApp/Cm/
+  license: BSD-4-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A multitask communication package
+  description: |
```

Comment:
This probably should be empty, either include a `description`, or remove the empty variable.

Reply:
I have added a short description of the package to this line.